src/components/dashboard/ClaimButton.js

```diff
@@ -51,10 +54,21 @@ const ButtonAmountToClaim = ({ showLabelOnly = false, entitlement, isCitizen, st
 )
 
 export const ButtonCountdown = ({ styles, nextClaim }) => (
-  <View style={styles.countdownContainer}>
-    <Text style={styles.extraInfoCountdownTitle} fontWeight="bold">
-      Your next daily claim:
-    </Text>
+  <View style={isSmallDev ? styles.countdownContainerSmallDev : styles.countdownContainer}>
```

Comment:
could we do this in withStyles HoC ? to return style that corresponds device size ?

Reply:
yes we can